pkgs/data/themes/arc/default.nix

```diff
@@ -11,19 +12,23 @@
 , cinnamon
 }:
 
-stdenv.mkDerivation rec {
+let
+  transparency = true;
+  gnomeShellResource = false;
```

Comment:
These aren't really useful as they can't be set by the user. They should either be put in the args list or just hardcoded.